src/ml_gym/frontend/dashboard/src/routes/throughput_board.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { IOStatsType, JobStatusType } from "../app/datatypes"
+
+type ThroughputProps = {
+    ioStats: IOStatsType
+};
+
+
+const getThroughput = (ioStats: IOStatsType, measurementDuration: number) => {
+    const currentTime = new Date().getTime()
+    const threshold = currentTime - measurementDuration * 1000
+    const msgTs = ioStats.msgTS
+
+    let i = msgTs.length - 1
+    while (i > 0 && msgTs[i] > threshold) {
+        i--
+    }
+    return ((msgTs.length - i - 1) / measurementDuration)
+};
+
+
+const Throughput: React.FC<ThroughputProps> = ({ ioStats }) => {
+
+    return (
+        <div id="throughput-board-container">
+            <h1> Throughput Board </h1>
+            <div>Connected: {ioStats.isConnected ? "yes" : "no"}</div>
+            <div>Messages received: {ioStats.msgTS.length}</div>
```

Comment:
Codacy has a fix for the issue: [Delete `······`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
      <div>Messages received: {ioStats.msgTS.length}</div>
```